### PR TITLE
Move ORFS docs to its own repo

### PR DESCRIPTION
**Pull request by habibayassin**
Signed-off-by: habibayassin <habibayassin@aucegypt.edu>

### CODE REVIEW

**Comment by maliberty** [comment]
has this been moved to ORFS?

**Comment by habibayassin** [comment]
> has this been moved to ORFS?

@maliberty yes, there is an open PR: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/673

File: docs/contrib/GettingInvolved.md [hunk 1]
```diff
@@ -3,11 +3,8 @@
 Thank you for taking the time to read this document and to contribute.
 The OpenROAD project will not reach all of its objectives without help!
 
-Possible ways to contribute:
+Possible ways to contribute in the scope of OpenROAD tool:
```
Comment (dralabeing) [inline docs/contrib/GettingInvolved.md:6]: Change to "Possible ways to contribute to the OpenROAD application:"

OpenROAD is the application name with underlying tools

**Review by dralabeing — COMMENTED** [review]
I am not entirely clear about the removal of sections from the OR document. Can you summarize what is the objective and what should I be reviewing?

**Comment by vvbandeira** [comment]
> I am not entirely clear about the removal of sections from the OR document. Can you summarize what is the objective and what should I be reviewing?

@dralabeing the goal of this PR and its counterpart in ORFS is to separate the documentation of ORFS and OR. Right now, the docs for both ORFS and OR are kept here in the OR repo, so whenever there's a change in ORFS, we actually need to create a PR here in OR to update the docs.

What we should be looking for in the PR review are any ORFS docs that should be moved (completely or by changing to a link), and also checking for wording/phrases that need to be adapted due to the change. I think the most important thing for this first PR is to make sure all docs are accounted for in one and only one repo. Other details and improvements can happen in future PRs.

File: docs/contrib/GettingInvolved.md [hunk 2]
```diff
@@ -23,20 +20,6 @@ philosophy. We do have exceptions in the project, but over time we hope
...
-In the
-[flow directory](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow)
-you will see a directory for
-[designs](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/designs)
-with Makefiles to run them, and one for PDK
-[platforms](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/platforms)
-used by the designs. If you add a new PDK platform, be sure to add at
-least one design that uses it.
-
 ## Contributing Scripts and Code
```
[the remark this answers is not in the document]
Comment (habibayassin) [reply docs/contrib/GettingInvolved.md:23]: @vvbandeira please advise here

File: docs/contrib/CI.md [hunk 3]
```diff
@@ -44,35 +44,6 @@ navigate Jenkins to access these features are available
...
-    -   Target: master branch.
--   OpenROAD-flow-scripts-Private [folder]
-    -   `public_tests_small`
-        -   Description: runs fast flow tests, does not include RTLMP designs.
-            Should finish in less than one hour.
-        -   Target: all branches that are not filed as PR. CI will run for PR
-            branches on the public side after "Ready to Sync Public" workflow.
--   OpenROAD-flow-scripts-All-Tests-Private
-    -   Description: runs flow tests, does not include RTLMP designs.
-    -   Target: secure branches.
```
Comment (vvbandeira) [inline docs/contrib/CI.md:73]: Let's replace with a link.

Comment (habibayassin) [reply docs/contrib/CI.md:73]: I assumed that the documentation link for ORFS will be this: https://openroadflow.readthedocs.io. Is this a correct assumption?

Comment (vvbandeira) [reply docs/contrib/CI.md:73]: It should be https://openroad-flow-scripts.readthedocs.io/en/latest/

File: docs/contrib/GettingInvolved.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vvbandeira) [reply docs/contrib/GettingInvolved.md:23]: We currently do not have any written instructions on code style for python (or Tcl) code, we can improve the docs on a separated PR later.

File: docs/toc.yml [hunk 4]
```diff
@@ -136,25 +134,5 @@ entries:
...
-    title: Metrics
-  - file: user/InstructionsForAutoTuner
-    title: AutoTuner
-
-
 - file: user/FAQS
   title: FAQs
-
-- file: tutorials/FlowTutorial
-  title: Flow Tutorial
```
Comment (vvbandeira) [inline docs/toc.yml:160]: Let's replace this with a new file called `tutorials/index.md`. In this new file, we should reference the ORFS dos tutorials and later we would be able to add OR-only tutorials.

File: docs/tutorials/index.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Tutorials
+
+## OpenROAD Flow Scripts Tutorial
+
+Flow tutorial can be access from OpenROAD Flow Scripts documentation [here](https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html)
```
Comment (vvbandeira) [inline docs/tutorials/index.md:5]:
```suggestion
Flow tutorial can be access from OpenROAD Flow Scripts documentation [here](https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html).
```

**Review by vvbandeira — APPROVED** [review]
